includes/definitions/adva_fsp150.yaml

```diff
@@ -0,0 +1,13 @@
+os: adva_fsp150
+text: ADVA_FSP150CC
```

Comment:
thx @laf. Remarks all understood and accepted. Only regarding the HW release I consider this as useful for the Ops people to immediately see which generation is in use. Will be back...